compiler-plugin/src/main/kotlin/arrow/meta/quotes/MetaExtensions.kt

```diff
@@ -248,7 +228,9 @@ fun Meta.parameterList(
   quote(match, map) { ParameterList(it) }
 
 /**
- * TODO
+ * ```
+ * """ $modifier $visibility val $name: $typeReference = $getter by $delegate""".property
+ * ```
```

Comment:
I'm starting to wonder if it makes sense to exclude all PSIElements from our scopes i.e. we can only get val/var from a property with this method in `KtProperty.java`:
```    @Override
    @NotNull
    public PsiElement getValOrVarKeyword() {
        PsiElement element = findChildByType(VAL_VAR_TOKEN_SET);
        assert element != null : "Val or var should always exist for property" + this.getText();
        return element;
    }
```

Reply:
We can wrap that method so it's safe or write a safe version of it directly over `KtProperty`

Reply:
Yes, I think in another PR, I want to go back to add more examples and do you suggest to wrap methods into safer versions.